### PR TITLE
Properly log action failure

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/Components/Action.vue [hunk 1]
```diff
@@ -34,31 +34,40 @@ export default {
...
-					OC.Notification.showTemporary(t('notifications', 'Failed to perform action'))
-				})
+				// do not do anything but log, the action went fine
+				// only the event bus listener failed, this is not our problem
+				} catch(error) {
+					console.error(error)
+				}
+			} catch(error) {
+				console.error('Failed to perform action', error)
+				OC.Notification.showTemporary(t('notifications', 'Failed to perform action'))
```
Comment (nickvergessen) [inline src/Components/Action.vue]: Should use the toast instead? @skjnldsv

Comment (skjnldsv) [reply src/Components/Action.vue]: Ah maybe, no clue :)

**Comment by skjnldsv** [comment]
what is this pr xD

**Comment by nickvergessen** [comment]
> what is this pr 

Merged. It is merged.

**Pull request merged**